app/services/valid_test_data_generators/applications_populater.rb

```diff
@@ -82,6 +83,20 @@ def create_participant(school:)
       reject_application(application)
     end
 
+    def create_participant_id_change(application)
+      return if Faker::Boolean.boolean(true_ratio: 0.3)
+
+      user = application.user
+      from_participant = application
```

Comment:
I did that originally but it messes up the participant numbers and causes another test to fail (if we want to create x number of participants, it would end up creating a random amount more). Instead I've opted to choose an existing/different participant that's associated with the same lead provider. It seems to work fine when I tested it?